sdk/vision/azure-ai-vision-imageanalysis/src/custom/custom.md

```diff
@@ -0,0 +1,164 @@
+# Java Image Analysis SDK Customization
+
+This document specifies the hand-customization done to the auto-generated Java Image Analysis SDK.
+
+## Add ImageBoundingBox.toString()
+
+Add the following method to the class ImageBoundingBox in file ImageBoundingBox.java:
+```java
+    /**
+     * Returns a compact string containing the bounding box values in a style suitable for printing.
+     *
+     * @return a string containing the bounding box values.
+     */
+    public String toString() {
+        return new StringBuilder().append("{x=").append(this.getX()).append(", y=").append(this.getY())
+            .append(", width=").append(this.getWidth()).append(", height=").append(this.getHeight()).append("}")
+            .toString();
+    }
+```
+
```

Comment:
Thanks for summarizing the partial-updated code! 
Happy to see the code are added to improve the user's usability(e.g. add overrides, or overwrite `toString()`), rather than adding workarounds.